tests/filecheck/dialects/arith/arith_ops.mlir

```diff
@@ -167,6 +167,10 @@
 
   // CHECK-NEXT: %cmpf = arith.cmpf ogt, %lhsf32, %rhsf32 : f32
 
+  %cmpf_fm = "arith.cmpf"(%lhsf32, %rhsf32) {"predicate" = 2 : i64, "fastmath" = #arith.fastmath<fast>} : (f32, f32) -> i1
+
+  // CHECK-NEXT: %cmpf_fm = arith.cmpf ogt, %lhsf32, %rhsf32 fastmath<fast> : f32
```

Comment:
This doesn't agree with the output of mlir (or at least the version of mlir we are compatible with).
```
 %58 = arith.cmpf ogt, %4#0, %4#1 {fastmath = #arith.fastmath<fast>} : f32
```
I see that it does agree with how this is parsed in mlir master though. Not sure what the correct thing to do here is.

Reply:
yeah `fastmath` should be in the properties dict in the generic format

Reply:
That too